Elevel/Application/Profiles/ReportProfile.cs

```diff
@@ -16,7 +13,9 @@ public ReportProfile()
             CreateMap<Report, GetReportListQuery.ReportDto>()
                 .ForMember(dest => dest.CoachName,
                     opt => opt.MapFrom(src =>
-                        src.Question.Creator.UserName.Replace("_", " ") ?? src.Audition.Creator.UserName.Replace("_", " ") ?? src.Topic.Creator.UserName.Replace("_", " ")))
+                        src.Question.Creator.UserName.Replace("_", " ") ??
```

Comment:
it's usually 
=> bla1
?? bla2
?? bla3

Reply:
Done